net/adguardhome/files/adguardhome.init

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+
+PROG=/usr/bin/AdGuardHome
+USE_PROCD=1
+
+# starts after network starts
+START=21
+# stops before networking stops
+STOP=89
+
+start_service() {
+  procd_open_instance
+  procd_set_param command "$PROG" -c /etc/adguardhome.yaml -w /tmp/adguardhome --no-check-update
```

Comment:
I see that the config file [will be created](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file) if it does not exist, but are there any sensible values we can provide in a default config file that would make setting up the service easier? (I haven't looked into this software enough to know if there are any default values that make sense.)

Reply:
The software generates the config file after first run. I can run AdGuardHome manually then copy the config file into repository. But IMHO it would be painful in maintenance.

Reply:
If there is only a partial config in the config file, is it smart enough to fill in the missing options with their default values? If so, then we can specify only the options we want to override; this should be easier to maintain.

Reply:
The thing is that when the config file isn't preset, the application starts setup wizard where you setup login/password, listening IPs/ports, etc. It would be hard to deploy a default config (especially with hardcoded user and encrypted password).
I've created a patch which makes AdGuardHome accept an empty configuration file. So, when the /etc/adguardhome.yaml is empty the application will start setup wizard. Hopefully this solution would be enough for you. Now, we have a config file (it's empty but at least it is present) inside the package archive.

Reply:
My point isn't to include an empty config file just to have the file present, it's to have sensible/usable defaults (even if partial, e.g. without username/password) for the user already set. (Also this is not a requirement, just a nice-to-have.) If this isn't currently possible then it's okay to not have this at this time.